src/js/actions/GuideActions.js

```diff
@@ -1,24 +1,29 @@
-import AppDispatcher from "../dispatcher/AppDispatcher";
-import GuideConstants from "../constants/GuideConstants";
-
-const GuideActions = {
-  /**
-   * @param {String} id to ignore
-   */
-  ignore: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_IGNORE, id
-    });
-  },
-
-  /**
-   * @param {String} id to follow
-   */
-  follow: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_FOLLOW, id
-    });
-  }
-};
-
-export default GuideActions;
+var Dispatcher = require("../dispatcher/Dispatcher");
```

Comment:
preference would be to `import` and `export`. If there is a reason not for doing so, I would say to use `let`